internal/impl/devbox.go

```diff
@@ -781,11 +783,22 @@ func (d *Devbox) computeNixEnv(ctx context.Context, usePrintDevEnvCache bool) (m
 	originalEnv := make(map[string]string, len(env))
 	maps.Copy(originalEnv, env)
 
+	var spinny *spinner.Spinner
+	if !usePrintDevEnvCache {
+		spinny = spinner.New(spinner.CharSets[11], 100*time.Millisecond, spinner.WithWriter(d.stderr))
+		spinny.FinalMSG = "✓ Got env-vars from nix.\n"
+		spinny.Suffix = " Getting env-vars from nix...\n"
```

Comment:
Computing new devbox environment...
...
Devbox environment computed

?

Reply:
Yeah, we could do that. I was trying to indicate to the user that this is nix being slow  (which it is, in this instance) 😃